zerver/models.py

```diff
@@ -2464,6 +2464,10 @@ class Stream(models.Model):
     }
     message_retention_days: Optional[int] = models.IntegerField(null=True, default=None)
 
+    can_remove_subscribers_group: Optional[UserGroup] = models.ForeignKey(
+        UserGroup, null=True, on_delete=models.RESTRICT
```

Comment:
Can you add a comment explaining the choice of RESTRICT here, since it isn't our default? (I think it might be correct, but seems good for posterity).